src/test/java/org/cactoos/text/TextOfTest.java

```diff
@@ -236,4 +238,51 @@ public void printsStackTrace() {
             )
         );
     }
+
+    @Test
+    public void readsFromInputStream() throws Exception {
+        final String before = "line1";
+        final InputStream stream = new ByteArrayInputStream(
+            before.getBytes(StandardCharsets.UTF_8.name())
+        );
+        MatcherAssert.assertThat(
+            "Can't read inputStream",
+            new TextOf(
+                stream
+            ).asString(),
```

Comment:
@iCok this can be on a single line: ``new TextOf(stream)``. Same for the other 2 tests.